### PR TITLE
Add exercise on classification metrics

### PR DESCRIPTION
**Pull request by ogrisel**
For the intro part.

### CODE REVIEW

File: content/python_files/classification_metrics.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
+# %% [markdown]
+#
+# # Classification metrics
+#
+# This notebook illustrate the impact of transforming predicted probabilities
```
Comment (glemaitre) [inline content/python_files/classification_metrics.py]:
```suggestion
# This notebook illustrates the impact of transforming predicted probabilities
```

File: content/python_files/classification_metrics.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+# %%
+def threshold_preserving_transformation(x):
+    return expit((x - 0.5) * 10)
+
+
+x = np.linspace(0, 1, 1000)
+y = threshold_preserving_transformation(x)
+_ = plt.plot(x, y)
+_ = plt.hlines(0.5, 0, 1, colors="gray", linestyles="--")
```
Comment (glemaitre) [inline content/python_files/classification_metrics.py]: Usually, I'm only catching the last line only. But use the style that you prefer and I'll make the rest of the notebooks consistent with it.

```suggestion
plt.plot(x, y)
plt.hlines(0.5, 0, 1, colors="gray", linestyles="--")
```

File: content/python_files/classification_metrics.py [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+#
+# - Using a thresholded metric to evaluate a probabilitic classifier does not
+#   inform us at all about the ability of the model to yield correct soft
+#   predictions. Hard classification metrics are very sensitive to the choice
+#   of the threshold. As a result, they are only meaningful if we also tune the
+#   threshold according to some business objective captured in the choice of
+#   the hard classification metric.
+# - Using a ranking loss to evaluate a probabilistic classifier only partially
+#   informs us about the ability of the model to yield correct soft
+#   predictions: ranking metrics
```
Comment (glemaitre) [inline content/python_files/classification_metrics.py]: It looks like the sentence is not finish ;)

Comment (ogrisel) [inline content/python_files/classification_metrics.py]:
```suggestion
#   predictions.
```

**Pull request merged**